src/test/e2e/00_use_cli_test.go

```diff
@@ -32,10 +32,10 @@ func TestUseCLI(t *testing.T) {
 		require.Contains(t, stdOut, string(b))
 	})
 
-	t.Run("zarf prepare sha256sum <local>", func(t *testing.T) {
```

Comment:
Not in scope for this, but should we start deprecating prepare? I was surprised to see it, needed to double check that the alias still existed.

Reply:
Yeah it would make sense to deprecate it. I've been using issues like this for deprecations - https://github.com/zarf-dev/zarf/issues/3420